edgedb/protocol/protocol.pyx

```diff
@@ -106,7 +106,7 @@ cdef class SansIOProtocol:
 
         self.server_settings = {}
         self.reset_status()
-        self.protocol_version = (PROTO_VER_MAJOR, PROTO_VER_MINOR_MIN)
+        self.protocol_version = (PROTO_VER_MAJOR, 0)
```

Comment:
Are you setting `minor` to `0` to force negotiation?

Reply:
No. Basically, because I don't know what to set. It should be never read before negotiation happened. Not sure setting it to None is a good idea...